src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> newList = new ArrayList<>();
+        for (int i = 0; i < 3; i++) {
```

Comment:
I think it would be better if you created a constant with a number of iterations.